dev/com.ibm.ws.jpa.tests.eclipselink_32_fat.common/bnd.bnd

```diff
@@ -0,0 +1,62 @@
+#*******************************************************************************
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+test.project: true
+publish.wlp.jar.disabled: true
+instrument.disabled: true
+
+# To define a global minimum java level for the FAT, use the following property.
+# If unspecified, the default value is ${javac.source}
+#fat.minimum.java.level: 17
+
+javac.source: 17
+javac.target: 17
+
+src: \
```

Comment:
Remove all the unused/not present directories... should only contain:

src: \
    test-applications/common/src